crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1026,7 +1027,14 @@ where
                                 self.record_process_transaction_metrics(tx_digest, &state);
                                 ReduceOutput::Success(result)
                             }
-                            Ok(None) => ReduceOutput::Continue(state),
+                            Ok(None) => {
+                                if state.non_retryable_stake >= validity_threshold {
```

Comment:
since Ok(None) can be returned if the aggregated sig verify failed and non_retryable_stake can be increased as a result of bad individual sig, this is to check against the threshold, if it is too high we should not retry.